src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -777,13 +759,19 @@
   <data name="ImageResizer_FilenameFormatPlaceholder.PlaceholderText" xml:space="preserve">
     <value>Example: %1 (%2)</value>
   </data>
-  <data name="PowerLauncher_Radio_Theme_Dark.Content" xml:space="preserve">
-    <value>Dark</value>
+  <data name="ColorModeHeader.Text" xml:space="preserve">
+    <value>Choose a mode</value>
   </data>
-  <data name="PowerLauncher_Radio_Theme_Default.Content" xml:space="preserve">
-    <value>System default</value>
+  <data name="Radio_Theme_Dark.Content" xml:space="preserve">
+    <value>Dark</value>
   </data>
-  <data name="PowerLauncher_Radio_Theme_Light.Content" xml:space="preserve">
+  <data name="Radio_Theme_Light.Content" xml:space="preserve">
     <value>Light</value>
   </data>
+  <data name="Radio_Theme_Default.Content" xml:space="preserve">
+    <value>Windows default</value>
+  </data>
+  <data name="Windows_Color_Settings.Content" xml:space="preserve">

```

Comment:
I would add the control type to the name:

```suggestion
  <data name="Link_Windows_Color_Settings.Content" xml:space="preserve">
```

Reply:
why?  no other control does that.

Reply:
1. The radio button names above have it.
2. I thought this makes it clear that this is a link name.

Reply:
As I said in the first comment, "I would suggest".
I am also fine with not adding the word "link".